points_name: "path.pgm"
map_name: "my_map.pgm"